### PR TITLE
Add new tab and section for displaying describe PVC output

### PR DESCRIPTION
**Pull request by reivaj05**
Closes: https://github.com/miniohq/engineering/issues/458
Tests will be implemented in: 830

### CODE REVIEW

**Review by jinapurapu — DISMISSED** [review]

File: portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      >
+      <h3>{title}</h3>
+    </Box>
+    
+  );
+};
+
+const PVCDescribeSummary = ({describeInfo}: IPVCDescribeSummaryProps) => {
+  return (
+    <React.Fragment>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx:121]: Please replace this to use Fragment only

```suggestion
    <Fragment>
```

File: portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  describeInfo: DescribeResponse;
+}
+
+interface IPVCDescribeAnnotationsProps {
+  annotations: Annotation[];
+}
+
+interface IPVCDescribeLabelsProps {
+  labels: Label[];
+}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx:82]: Please define this inside a types file

File: portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+          <LabelValuePair label={"Volume"} value={describeInfo.volume} />
+          <LabelValuePair label={"Volume Mode"} value={describeInfo.volumeMode} />
+        </Box>
+    </React.Fragment>
+  );
+};
+
+const PVCDescribeAnnotations = ({annotations}: IPVCDescribeAnnotationsProps) => {
+  return (
+    <React.Fragment>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/pvcs/PVCDescribe.tsx:140]: Same case for this
```suggestion
    <Fragment>
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by prakashsvmx — APPROVED** [review]
Tested. Changes look good to me. :+1:

**Review by dvaldivia — APPROVED** [review]
tested. LGTM

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**